Tests/Marketplace/upload_packs.py

```diff
@@ -217,17 +217,17 @@ def clean_non_existing_packs(index_folder_path: str, private_packs: list, storag
     Returns:
         bool: whether cleanup was skipped or not.
     """
-    if ('CI' not in os.environ) or (
-            os.environ.get('CI_COMMIT_BRANCH') != 'master' and storage_bucket.name == GCPConfig.PRODUCTION_BUCKET) or (
-            os.environ.get('CI_COMMIT_BRANCH') == 'master' and storage_bucket.name not in
-            (GCPConfig.PRODUCTION_BUCKET, GCPConfig.CI_BUILD_BUCKET)):
-        logging.info("Skipping cleanup of packs in gcs.")  # skipping execution of cleanup in gcs bucket
-        return True
```

Comment:
Can you elaborate on what is this part you removed here?

Reply:
I removed the code that makes it not run in tests (it was part of the issue)